stock_move_backdating/__openerp__.py

```diff
@@ -19,22 +19,34 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
+
+
 {
-    "name": "Stock Move Backdating",
-    "version": "1.0",
-    'author': ['Marco Dieckhoff, BREMSKERL', 'Agile Business Group'],
-    "category": "Stock Logistics",
+    'name': 'Stock Move Backdating',
+    'version': '1.0',
+    'category': 'Stock Logistics',
+    'description': """
+Allows back-dating of stock moves
+=================================
+This module allows to register old stock moves (with date != now).
+On stock moves, user can specify the "Actual Movement Date", that will be
+used as movement date.
+
+Contributors
+------------
+* Marco Dieckhoff
+* BREMSKERL
+* Agile Business Group
+* El Hadji Dem (elhadji.dem@savoirfairelinux.com)
+* David Dufresne (david.dufresne@savoirfairelinux.com)
+    """,
+    'author': 'Bremskerl',
```

Comment:
could you add "Odoo Community Association (OCA)" to the authors, so that the module shows up under the association on app.odoo.com?